status/src/server.ts

```diff
@@ -0,0 +1 @@
+export const server = 'peer.decentraland.org'
```

Comment:
If we change this to:
```
process.ENV.CATALYST_URL ?? peer.decentraland.org
```
then we can easily run this together with catalysts. 

Take into account that `CATALYST_URL` includes the protocol (http or https)

Reply:
Changed to be REACT_APP_CATALYST_URL